docs/reference/controls/colorpicker/README.md

```diff
@@ -0,0 +1,113 @@
+# ColorPicker
```

Comment:
File header also can be added, see the same button page as a reference. But that's optional with this docs engine.

More importantly, `builtin-controls.md` file should be updated to include ColorPicker as well. 
It seems like we don't need to update sidebar.js for this tho.

Reply:
Also made both recommended changes